zerver/webhooks/airbyte/view.py

```diff
@@ -0,0 +1,90 @@
+# Webhooks for external integrations.
+
+from django.http import HttpRequest, HttpResponse
+
+from zerver.decorator import webhook_view
+from zerver.lib.response import json_success
+from zerver.lib.typed_endpoint import JsonBodyPayload, typed_endpoint
+from zerver.lib.validator import WildValue, check_bool, check_int, check_string
+from zerver.lib.webhooks.common import check_send_webhook_message
+from zerver.models import UserProfile
+
+AIRBYTE_TOPIC_TEMPLATE = "{workspace} - {connection} - {source} - {destination}"
+
+AIRBYTE_MESSAGE_TEMPLATE = """
+Airbyte job {jobId} **{status}** in {durationFormatted}.
+
+Connection: [{connection_name}]({connection_url})
+**Details:**
```

Comment:
I'd drop this header and the job ID. Maybe simplify a bit and add an emoji green/red circle...

Example Message:

> :green_circle: Airbyte sync **succeeded** for [connection name link]().
> 
> - **Source**: [link]()
> - **Destination**: [link]()
>  - **Duration**:  time
> - **Records**: data
> - **Bytes**: data

Reply:
Green if it is successful and red if the sync failed?

Reply:
So, should take the duration out from the successful message and just add it as part of the details?

Reply:
Yeah, you can use the emoji strings for the two circles: `":green_circle:"` and `""red_circle"`.

And I think the duration would be better as a bullet point in the list. I think the more important part of the success/fail message is the link to the connection that's set up in Airbyte and the status.